app/static/src/css/custom.css

```diff
@@ -1,5 +1,5 @@
 .govuk-footer__copyright-logo {
-    background-image: url("/static/images/govuk-crest.png")
+    background-image: url("/static/govuk-frontend/images/govuk-crest.png")
```

Comment:
Nice one adding this back. The figma also includes this:
![image](https://github.com/communitiesuk/funding-service-design-post-award-submit/assets/22678716/450f116e-db24-41c6-89b4-13b44b2ab98e)
Can we also add this icon as well?

Reply:
Added this